File: Algorithmique appliquée/chapitre 1/python/exercice3.py
# -*-coding:utf-8 -*
# Programme exercice 3
# somme de nombres entre 2 bornes
# date : 15/09/2016


import os


somme = 0

# saisie des bornes
min = int(input("entrez la valeur min = "))
max = int(input("entrez la valeur max = "))

# calcul et affichage de la somme
for k in range(min, max+1) :
    somme += k
print("La somme des valeurs est = ", somme)


os.system("pause")

File: Algorithmique appliquée/chapitre1/cpp/exercice3.cpp
/*  programme somme(exercice3)
*   but: somme de nombres entre deux bornes
*   date: 15/09/2016
*/
#include <stdio.h>
#include <iostream>
using namespace std;

int main(int argc, char *argv[]){

    unsigned int min, max, k,somme =0;

    cout << "entrez les bornes: \n";
    cin >> min >> max;

    for(k=min;k < max;k++){
        somme += k;
    }
    cout << "La somme des valeurs est = " << somme;
    return 0;
}
